infrastructure/code/template/details.html

```diff
@@ -23,17 +23,70 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <style>
     :root {
-      --text-color: #d4d4d4;
+      /* JetBrains Typography */
+      --text-color-light: #000000;
```

Comment:
How are these values updated when a theme changes?

Reply:
This PR goes together with the changes for [IntelliJ](https://github.com/snyk/snyk-intellij-plugin/pull/514).

- When IntelliJ renders the template served by Language Server, it also injects custom scrips.
- One of these scripts is for overriding CSS custom properties.
- Here, in the Language Server template, we define CSS global variables like the above, but let's use `--example-line-removed-color` to show how it works:

1. Define the themed variables colors in `:root` element:

```html
  <style>
    :root {
...
      --example-line-removed-color-light: #F5E6E7;
      --example-line-removed-color-dark: #352628;
      --example-line-removed-color-darcula: #563E3F;
    }
...

``` 

2. Set the background color for the DOM element `example-line.removed` with the variable name without the theme part: `--example-line-removed-color`

```html
...
    .example-line.removed {
      background-color: var(--example-line-removed-color);
    }...
  </style>
``` 

3. Then in **IntelliJ** we inject the `--example-line-removed-color` with the value retrieved from `--example-line-removed-color-darcula` if `isDarculaTheme === true` for example:

```kotlin
fun generate(jbCefBrowser: JBCefBrowserBase): CefLoadHandlerAdapter {
  // ...
  override fun onLoadEnd(browser: CefBrowser, frame: CefFrame, httpStatusCode: Int) {
    if (frame.isMain) {
      val isDarculaTheme = isDarcula()
      val isDarkTheme = EditorColorsManager.getInstance().isDarkEditor
      val themeScript = """
        (function(){ 
          if (window.themeApplied) { return; }
          window.themeApplied = true;
          const style = getComputedStyle(document.documentElement);
          // ⬇️
          document.documentElement.style.setProperty('--example-line-removed-color', style.getPropertyValue('--example-line-removed-color-${if (isDarkTheme) "dark" else "light"}'));

          if (${isDarculaTheme}) {
              document.documentElement.style.setProperty('--example-line-removed-color', style.getPropertyValue('--example-line-removed-color-darcula'));
          }
        })();
        """
      browser.executeJavaScript(themeScript, browser.url, 0)
  }
}
```

4. So when the function `browser.executeJavaScript(themeScript, browser.url, 0)` executes, the property `--example-line-removed-color` will contain the value of `--example-line-removed-color-darcula` 😃

Reply:
Let's support all possible themes, as discussed. We'll prioritize that as an incremental add-on.